### ISSUE TITLE
High elves feel overshadowed by other spellcasting races

### ISSUE THREAD

**Issue by FraterFugae** [jeremygurr/dcssca#409]
As they are right now, high elves have nothing interesting about them that set them apart from other, more unique spellcasters. While they do have slightly different aptitudes, lower health and higher mana, their differences are not as noticeable as on other species (like the huge deep elf mana pool, or the easy-but-weak sludge elf spells). They usually end up playing like an underwhelming humanoid hybrid, and just don't feel as fun as others. I brainstormed a few ideas for how to set them apart, could you give me some feedback on them?
1. Make their aptitude differences more drastic, with higher skill in elemental magics (ideally ones that deep elves or other species don't cover)
2. Treat them like "magical humans," giving them better XP aptitude and flattening their magic to +1.
3. Give them a mutation that makes them desirable over other species, like a spellpower bonus (though that may be too op) or extra spell levels.
4. Theme them around being "acolytes/clerics of magic," giving them more piety and/or stronger abilities under the magic gods, plus better invocations skill

I understand that there is a place for weaker species, especially if they offer unique gameplay (like octopodes!), but I don't feel like high elves are unique enough for how boring they are.

**Comment by Sandman25DCSS** [comment]
I agree that High Elves are boring. I always wondered if we can have a species with Nightstalker mutation in order to avoid scumming Demonspawn. Tornado is extremely fun with Nightstalker 3 and High Elf has one of the best aptitude in Air Magic. Though this is probably OP so maybe give them an ability to evoke Darkness spell instead.

**Comment by FraterFugae** [comment]
Nightstalker seems like something more suited to sludge elves thematically, since they use "dark magic." I do agree, though, that that would make a really interesting magic race.

Maybe give high elves easier spells and sludge elves nightstalker?

**Comment by Sandman25DCSS** [comment]
If you mean [ "Subdued magic"](http://crawl.chaosforge.org/Bad_mutations#Subdued_Magic) as easier spells, then it is a bad mutation. Maybe Wild magic 1 at XL 10?  That should be different enough from Halflings with their Wild Magic 3.

**Comment by jeremygurr** [comment]
Totally agree with the need to make HE more interesting. I've struggled with this one. 

The only issue with giving nightstalker to sludge elves is that you couldn't easily pair it with tornado. Although SE could cast tornado easily enough, it's power levels would be quite low. I tried tornado on an SE and it was underwhelming (which is part of the point of SE). 

Giving high elves an inherent darkness ability would definitely make them more interesting, and yet still be a little different than the permanent darkness of nightstalker. That could be part of the high elves lore, that they wanted the world to leave them alone so they advanced their magic until they became masters of hiding. 

There is one more thing that we could do to distinguish them. I think it would be cool to have a magic race that specialized in elemental magic. All of their elemental schools would be +2. But all of their other magic schools would be -2, except for spellcasting and conjurations, which remain at the +1 they are at now.

**Comment by yrmvgh** [comment]
> I always wondered if we can have a species with Nightstalker mutation in order to avoid scumming Demonspawn

Interestingly enough, "Nightstalker species" is already something that has a place (somewhere) on the endless to-do list.

Slimeoid (amorphous, eats junk), nightstalker (nightcrawler), fairy (10 hp, tiny, but each source of damage only deals 1 hp even if it's a spiked club), and maybe troglodytes (big stinky humanoids from the frozen north!).

One of the things we've talked about both on github and IRC, is whether there are any races that can be axed to "make room" for additional races. We have the two orcs, the three variations of elves, and I've never personally witnessed anyone playing centaurs, nagas, or halflings on the server - though it has probably happened at least once.

If we don't axe one of the elves in the elfpile though, I'd like to make deep elves more closely-entwined with or reliant on stealth, high elves more reliant on dodging, and sludge elves more reliant on armour. Also, increase/decrease inv/evo for HE so it's more Sif-friendly, and increase/decrease evo/inv for DE so it's more crystal-ball friendly.

**Comment by Sandman25DCSS** [comment]
Elemental race is not needed, high Int and good aptitude in Conjurations allow to cast everything you want with a small investment in elemental schools. Just take my last game (http://crawl.homedns.org/morgue/Sandman25/morgue-Sandman25-20160502-234131.txt):

```
 - Level 22.8 Conjurations
 - Level 21.9 Fire Magic
 - Level 7.8 Earth Magic
   Level 9.0 Poison Magic
       Poison Arrow      |       |       |       |    22 |   243 |   406 |   281 |   428 |   273 ||  1653
       Lee's Rapid Decon |       |       |       |       |       |    11 |     6 |     6 |       ||    23
       Iron Shot         |       |       |       |       |       |       |       |       |   260 ||   260
      Fire Storm        |       |       |       |       |       |       |       |     5 |   369 ||   374

```

Also CA fork made Haste quite bad, I barely casted it
       Haste             |       |       |       |       |       |       |     2 |       |     5 ||     7
Translocations don't have decent spells at level 5-7 and summoning is not really needed when you are killing monsters with Conjurations

**Comment by Sandman25DCSS** [comment]
I did win Na and Ha in CA fork...
I will try Ce, it should be fun/challenging without extra speed.

**Comment by FraterFugae** [comment]
Of course, there's always the option to just remove high elves, since there are already a couple of other species that do "magic/melee hybrid" better. I would be sad to see them go, though, since i really liked them in vanilla, and it feels like there's potential in having a "middle of the road" humanoid caster with no real racial armour/god restrictions.
